OneCommunityGlobal/HighestGoodNetworkApp
# Obeda lint fix Common component

# Description
Lint Fix for Common Component `src/components/common/**`

## Main changes explained:
- Executed the link checker
- Fixed changes manually apart from fix done by lint--fix
- Removed the component from .prettierignore and .eslintignore

## How to test:

1. check into current branch
2. Run `npm install` and `npx eslint src/components/common/*` to make sure there is no lint errors
3. Run `npm run test` to make sure that all tests pass
4. Run `npm run start:local`
5. log as any user
6. Verify that common components such as checkbox, form, dropdown, etc work properly 

## Screenshots or videos of changes:
![successful lint fix common component](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/61763829/c0931cf9-94c0-46c0-9426-80eb232b7fc1)
